app/controllers/home_controller.rb

```diff
@@ -0,0 +1,22 @@
+class HomeController < ApplicationController
+  before_action :get_movies
```

Comment:
Generally I like the use of `before_action` (as evidenced by my suggestion for `albums_controller.rb`), but here I'm not sure that we need it.
We've only got one action in `HomeController`, so the use of `before_action` is mostly just spreading out the lines that go in the `index` method, without reducing code duplication.